modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/Server.java

```diff
@@ -79,6 +80,11 @@
      */
     private static final int BLOCKING_TASK_EXECUTOR_THREADS = 32;
 
+    /**
+     * Socket receive and send buffer size in bytes.
+     */
+    private static final int SOCKET_BUFFER_SIZE = 1048576;
```

Comment:
Will it be better if we can make this configurable?

Reply:
:+1: